### PR TITLE
Added Residual of Residual Networks 

### PR DESCRIPTION
**Pull request by titu1994**
Added Residual of Residual Networks to applications folder.

### CODE REVIEW

File: keras_contrib/applications/ror.py [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+
+    return model
+
+
+def __initial_conv(input):
+    x = Convolution2D(16, 3, 3, border_mode='same', init='he_normal')(input)
+    return x
+
+
+def __conv1_block(input, k=1, dropout=0.0, initial=False):
```
Comment (kemaswill) [inline keras_contrib/applications/ror.py]: It seems that __conv1_block(), __conv2_block() and __conv3_block() only differs in the number of feature maps, i.e., 16, 32, 64. Could you merge these 3 functions to 1 and set the number of feature maps as a parameter?

File: examples/ror_cifar10.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+print("Finished compiling")
+
+model.fit_generator(generator.flow(trainX, trainY, batch_size=batch_size), samples_per_epoch=len(trainX),
+                    nb_epoch=nb_epoch,
+                    callbacks=[callbacks.ModelCheckpoint("weights/RoR-WRN-40-2-Weights.h5", monitor="val_acc",
+                                                         save_best_only=True, save_weights_only=True)],
+                    validation_data=(testX, testY),
+                    nb_val_samples=testX.shape[0], verbose=2)
+
+yPreds = model.predict(testX)
```
Comment (kemaswill) [inline examples/ror_cifar10.py]: Model should be evaluated using model.evaluate(X_test, Y_test). Please add the evaluation results as comments in the beginning of this script.

File: keras_contrib/applications/ror.py [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+        nb_classes: number of classes
+        img_input: tuple of shape (channels, rows, columns) or (rows, columns, channels)
+        include_top: flag to include the final Dense layer
+        depth: depth of the network
+        width: width of the network
+        dropout: Adds dropout if value is greater than 0.0.
+                 Note : Generally not used in RoR
+
+    Returns: a Keras Model
+
```
Comment (kemaswill) [inline keras_contrib/applications/ror.py]: Delete empty line here.

File: keras_contrib/applications/ror.py (hunk 1 above)
Comment (titu1994) [inline keras_contrib/applications/ror.py]: Hmm, I would normally agree, but the paper describes it in this format. Well, should I call the method __conv_block(input, i, k=1, dropout=0.0, initial=False) and then multiple i with 16 everywhere?

**Comment by kemaswill** [comment]
Please change the file name and add the performance in the beginning.

File: keras_contrib/applications/ror.py (hunk 1 above)
Comment (titu1994) [inline keras_contrib/applications/ror.py]: Actually, model doesn't build correctly if I remove it. It is simpler just to keep it.

**Comment by titu1994** [comment]
Oh crap. I'm gonna close this PR. It's merged with old changes and created a mess.

**Pull request closed without merging**